assets/src/edit-story/components/canvas/carousel/compactIndicator.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+function CompactIndicatorWithRef({ onClick, isActive, ariaLabel }, ref) {
+  return (
+    <Indicator
+      onClick={onClick}
+      isActive={isActive}
+      aria-label={ariaLabel}
+      role="option"
+      ref={ref}
+    />
+  );
+}
+
+const Indicator = styled.button`
+  display: block;
+  width: 48px;
+  height: 4px;
+  margin: 0 8px 0 0;
+  &:last-of-type {
```

Comment:
Each indicator is wrapped in a `<li>` in the carousel, so this is always true. I'd generally set `margin: 0` and not use `:last-of-type`.

Reply:
Correct, this should be removed, last minute I switched from `Li | Fragment` wrapper, right now it's already handled in that parent `Li` commented above.